packages/transporter/src/createStatelessHost.ts

```diff
@@ -1,6 +1,14 @@
 import { CallEnum, HostOptions, StatelessHost } from '.';
 
 export function createStatelessHost(options: HostOptions): StatelessHost {
+  if (typeof options === 'string') {
+    return {
+      protocol: 'https',
```

Comment:
Do we need to force the protocol? I think I would rather keep not set the `protocol` key and let the regular business logic deal with it.

Reply:
this is the only place where protocol is set. If you set `{ url: xxx }` the protocol will also be https. If you want a custom protocol you can use the object form as normal. Don't you think that most cases would be using https?

Reply:
I think `https` is a reasonable default here.
Also FMPOV the DX is better if `createStatelessHost({url: xxx})` and `createStatelessHost(xxx)` have the same behavior